packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -296,6 +303,40 @@ const getSharingPolicyForFile = (document, sharingType) => {
     : { update: 'push', remove: 'revoke' }
 }
 
+const getSharingRulesForOrganizations = document => {
```

Comment:
Let's write a JSdoc. Let's prepare the futur ;)

Reply:
Done, I documented only the added function. Others functions have been documented in a separate PR : https://github.com/cozy/cozy-client/pull/994